tasks/system/Linux.yml

```diff
@@ -23,6 +17,10 @@
       state: "present"
       home: "/home/{{ sonar_user }}"
       createhome: True
+    notify: Stop Sonarqube
+
+  - name: "Force all notified handlers to run"
+    meta: flush_handlers
```

Comment:
Why handlers?

Reply:
Starting/stopping with service task breaks idempotence test. Do you have ideas how to fix it without handlers?